apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -537,6 +538,63 @@ describe('Trigger event - /v1/events/trigger (POST)', function () {
     expect(message).to.be.null;
   });
 
+  it('should use Novu integration for new orgs', async function () {
+    const old = await integrationRepository.find({
+      _organizationId: session.organization._id,
+      _environmentId: session.environment._id,
+    });
+
+    for (const oldKey in old) {
+      await integrationRepository.delete({
+        _id: old[oldKey]._id,
+        _organizationId: session.organization._id,
+        _environmentId: session.environment._id,
+      });
+    }
```

Comment:
```suggestion
      await integrationRepository.delete({
        _id: { $in: existingIntegrations.map(integration => integration._id)},
        _organizationId: session.organization._id,
        _environmentId: session.environment._id,
      });
    }
```

A very minor suggestion, but could be also represented in a single call